codegen/chainconfig.go

```diff
@@ -134,4 +134,10 @@ var ChainConfigByID = map[string]*ChainConfig{
 		FirehoseEndpoint: "mainnet.stellar.streamingfast.io:443",
 		Network:          "stellar-mainnet",
 	},
+	"sei-mainnet": {
+		DisplayName:      "Sei Mainnet",
```

Comment:
How to they write it, `SEI Mainnet` or `Sei Mainnet`?

Reply:
In the documentation, they always seem to use `Sei`
https://www.docs.sei.io/learn/general-overview